addon/models/node.js

```diff
@@ -25,10 +35,12 @@ export default DS.Model.extend(OsfModel, {
         inverse: 'children'
     }),
     children: DS.hasMany('nodes', {
-        inverse: 'parent'
+        inverse: 'parent',
+        updateRequestType: 'POST'
     }),
     affiliatedInstitutions: DS.hasMany('institutions', {
-        inverse: 'nodes'
+        inverse: 'nodes',
+        serializer: relationshipSerializer('affiliatedInstitutions', 'institutions')
```

Comment:
Why do institutions use a custom serializer, but not children? Can we add any documentation to clarify the very special, complicated rules in play for these relationships?